app/helpers/application_helper.rb

```diff
@@ -1,2 +1,18 @@
 module ApplicationHelper
+  def bootstrap_class_for(flash_type)
+    { success: "alert-success", error: "alert-danger", alert: "alert-warning", notice: "alert-info" }[flash_type.to_sym] || flash_type.to_s
+  end
+
+  def flash_messages(opts = {})
+    flash.each do |msg_type, message|
+      concat(content_tag(:div, message, class: "alert #{bootstrap_class_for(msg_type)} alert-dismissible", role: 'alert') do
+        concat(content_tag(:button, class: 'close', data: { dismiss: 'alert' }) do
```

Comment:
Line is too long. [82/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.